docs/managing-cpp-deps.md

```diff
@@ -0,0 +1,48 @@
+---
+id: managing-cpp-deps
+title: Managing C++ dependencies
+---
+
+Details to consider when consuming a Visual C++ project as a source dependency of your app.
+
+### Details
+
+Applications with native code, either written on `C#` or `C++`, may add source dependencies on native Visual C++ (`.vcxproj`) projects.\
+Meaning, such dependencies will be built as part of the application.
+
+Starting with version `0.68`, React Native for Windows apps use the [`PackageReference`](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files) restore project style for native `C++` NuGet dependencies.\
+The main change consists in NuGet packages being directly loaded from the user account's `globalPackagesFolder` cache instead of also copying them into a local folder relative to the Visual Studio Solution location (see [`repositoryPath`](https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file)).
+
+This may conflict with Node/Visual C++ dependencies that use the more common [`packages.config`](https://docs.microsoft.com/en-us/nuget/reference/packages-config) project style (i.e. [React Native Picker](https://github.com/react-native-picker/picker#react-native-pickerpicker)).
```

Comment:
There are many community modules, so the reader might be using a different one, and not be familiar with Picker. 

```suggestion
This may conflict with Visual C++ dependencies that use the more common [`packages.config`](https://docs.microsoft.com/en-us/nuget/reference/packages-config) project style, including community modules generated targeting versions before 0.68.
```

Reply:
Same feedback for below, for anywhere we reference anything specific to Picker.